databricks/koalas/generic.py

```diff
@@ -1369,29 +1369,60 @@ def median(self, accuracy=10000):
 
         >>> df['a'].median()
         25.0
+        >>> (df['a'] + 100).median()
+        125.0
+
+        For multi-index columns,
+
+        >>> df.columns = pd.MultiIndex.from_tuples([('x', 'a'), ('y', 'b')])
+        >>> df
+              x  y
+              a  b
+        0  24.0  1
+        1  21.0  2
+        2  25.0  3
+        3  33.0  4
+        4  26.0  5
+
+        On a DataFrame:
+
+        >>> df.median()
+        x  a    25.0
+        y  b     3.0
+        Name: 0, dtype: float64
+
+        On a Series:
+
+        >>> df[('x', 'a')].median()
+        25.0
+        >>> (df[('x', 'a')] + 100).median()
+        125.0
         """
         if not isinstance(accuracy, int):
             raise ValueError("accuracy must be an integer; however, got [%s]" % type(accuracy))
 
         from databricks.koalas.frame import DataFrame
-        from databricks.koalas.series import Series
+        from databricks.koalas.series import Series, _col
 
         kdf_or_ks = self
         if isinstance(kdf_or_ks, Series):
-            ks = kdf_or_ks
-            return self._reduce_for_stat_function(
+            ks = _col(kdf_or_ks.to_frame())
+            return ks._reduce_for_stat_function(
```

Comment:
i don't think it is important, but maybe we better use `kser` rather than `ks` when indicate `Series` since the `ks` is used when import koalas conventionally like `import databricks.koalas as ks` ?

Although maybe it is really not so important, I mentioned it because i thought you think about these conventions are important when you work like #919

Reply:
Good catch! Yeah we should use `kser` here.